celestia/peers.txt

```diff
@@ -1 +0,0 @@
-
```

Comment:
Can you simply not modify this file please?

Reply:
Hey, I simply followed the announcement from Discord. But let me add it to the peers file instead.

Reply:
The discord announcement clearly states to create a separate file and not modify the existing celestia/peers.txt